Philips.CodeAnalysis.Common/DiagnosticIds.cs

```diff
@@ -110,5 +110,6 @@ public enum DiagnosticIds
 		AvoidMultipleLambdasOnSingleLine = 2115,
 		AvoidArrayList = 2116,
 		AvoidUnnecessaryWhere = 2117,
+		DontUseMagicNumbers = 2118,

```

Comment:
Just for consistency, rename to AvoidMagicNumbers

Reply:
Done